opencti-platform/opencti-graphql/src/database/file-storage.js

```diff
@@ -306,18 +306,18 @@ export const upload = async (context, user, path, fileUpload, opts) => {
 
 export const filesListing = async (context, user, first, path, entity = null, prefixMimeType = '') => {
   const filesListingFn = async () => {
-    const files = await rawFilesListing(context, user, path);
+    let files = await rawFilesListing(context, user, path);
+    if (entity) {
+      files.filter((file) => file.metaData.entity_id === entity.internal_id);
```

Comment:
You need to assign the return of filter method
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/filter